src/net/rpc.rs

```diff
@@ -103,6 +103,7 @@ pub const STREAM_CHUNK_SIZE: u64 = 4096;
 #[derive(Default)]
 pub struct RPCHandlerArgs<'a> {
     pub exit_at_block_height: Option<&'a u64>,
+    pub genesis_chainstate_hash: String,
```

Comment:
This should be a `util::hash::Sha256Sum` struct, which can be constructed with `Sha256Sum::from_hex(&str)`

Reply:
This and the related feedback should all be fixed now.